tgui/packages/tgui/interfaces/SkillsMenu.js

```diff
@@ -0,0 +1,75 @@
+import { capitalize } from 'common/string';
+import { Fragment } from 'inferno';
+import { useBackend } from '../backend';
+import { Section, Box, ProgressBar, Slider, NoticeBox, Button } from '../components';
+import { Window } from '../layouts';
+
+export const SkillsMenu = (props, context) => {
+  const { act, data } = useBackend(context);
+  const { skillset_name, skills, owner, admin } = data;
+
+  return (
+    <Window width={300} height={550}>
+      <Window.Content>
+        <Section
+          title={'Skillset name: ' + skillset_name}
+          buttons={
+            <Button
+              color="transparent"
+              tooltip="Refresh"
+              tooltipPosition="left"
+              icon="sync-alt"
+              onClick={() => act('refresh')}
+            />
+          }>
+          {skills ? (
+            admin ? (
+              <SkillsEdit />
+            ) : (
+              <SkillsView />
+            )
+          ) : (
+            <NoticeBox danger>Warning! Mob has null skills!</NoticeBox>
+          )}
+        </Section>
+      </Window.Content>
+    </Window>
+  );
+};
+
+const SkillsView = (props, context) => {
+  const { act, data } = useBackend(context);
+  const { skillset_name, skills, owner, admin } = data;
+  return skills.map((skill, index) => (
+    <Fragment key={index}>
+      <ProgressBar value={skill.level / skill.maxlevel}>
+        {capitalize(skill.name)}: {skill.level}/{skill.maxlevel}
```

Comment:
```suggestion
        {capitalizeAll(skill.name)}: {skill.level}/{skill.maxlevel}
```
this prevents JTAC being converted to Jtac and makes sure Illegal Technology etc is properly uppercased